MedicationScheduler/AuthApp/templates/profile.html

```diff
@@ -0,0 +1,143 @@
+<!--https://www.bootdey.com/snippets/view/account-settings#html-->
+
+{% extends "base.html" %}
+{% block content %}
+
+<div class="container light-style flex-grow-1 container-p-y">
+
+    <h4 class="font-weight-bold py-3 mb-4">
+      Account settings
+    </h4>
+
+    <div class="card overflow-hidden">
+      <div class="row no-gutters row-bordered row-border-light">
+        <div class="col-md-3 pt-0">
+          <div class="list-group list-group-flush account-settings-links">
+            <a class="list-group-item list-group-item-action active" data-toggle="list" href="#account-general">General</a>
+            <a class="list-group-item list-group-item-action" data-toggle="list" href="#account-change-password">Change password</a>
+            <a class="list-group-item list-group-item-action" data-toggle="list" href="#account-info">Info</a>
+            <a class="list-group-item list-group-item-action" data-toggle="list" href="#caregiver">Caregiver</a>
+            <a class="list-group-item list-group-item-action" data-toggle="list" href="#account-notifications">Notifications</a>
+          </div>
+        </div>
+        <div class="col-md-9">
+          <div class="tab-content">
+            <div class="tab-pane fade active show" id="account-general">
+              <hr class="border-light m-0">
+              <div class="card-body">
+                <form method="post">
+                      {% csrf_token %}
+                      {{ user_update_form.as_p }}
+                      <button type="submit" class="btn btn-primary" name="user_update">Save changes</button>
+                </form>
+              </div>
+            </div>
+            <div class="tab-pane fade" id="account-change-password">
+              <div class="card-body">
+                    <form method="post">
+                      {% csrf_token %}
+                      {{ password_change_form.as_p }}
+                      <button type="submit" class="btn btn-primary" name="change_password">Save changes</button>
+                    </form>
+              </div>
+            </div>
+            <div class="tab-pane fade" id="account-info">
+              <div class="card-body">
+                <form method="post">
+                      {% csrf_token %}
+                      {{ update_profile_form.as_p }}
```

Comment:
I can't leave a field empty in the update profile form.

Reply:
done

Reply:
@kdgromova looks like everything is working except when I leave the birthday field empty I get the following error. I'm not sure why:
![image](https://user-images.githubusercontent.com/82332510/149668222-1a689663-da27-451a-9774-5ad823e424b0.png)

Reply:
@bew5294 that's very strange. It works on my end.

Reply:
oh, now I got this problem.

Reply:
@bew5294 I fixed it